amt/wsman.py

```diff
@@ -36,12 +38,13 @@
     'cd': 'Intel(r) AMT: Force CD/DVD Boot',
 }
 
-
+FRIENDLY_POWER_STATE = {v: k for (k, v) in six.iteritems(POWER_STATES)}
```

Comment:
I'm fine with the optimization, to build the reverse map in advance, however using six.iteritems is really overkill. POWER_STATES.items() is fine and less ugly.